Source/WebCore/platform/network/ResourceResponseBase.cpp

```diff
@@ -60,6 +60,16 @@ ResourceResponseBase::ResourceResponseBase()
 {
 }
 
+ResourceResponseBase::ResourceResponseBase(URL&& url, String&& mimeType, long long expectedLength, const String& textEncodingName)
```

Comment:
Should we use `String&& textEncodingName`?

Reply:
In the callsites I found, there were cases where the text encoding was a const reference of something local, so we wouldn't be able to move it. So I left that argument alone.

We could add another constructor with a move version, which might be an improvement for the serializer (which moves all arguments). I could do a follow-up with that.